src/script/view_model/CallingViewModel.ts

```diff
@@ -88,7 +88,7 @@ declare global {
     setSinkId?: (sinkId: string) => Promise<void>;
   }
 }
-
+const maxGroupSize = 4;
```

Comment:
Is it really 4? 😅

Reply:
Yes. Currently on iOS we show a dialog when a user is initiating a call with more than 4 people and we need to implement the same for Web.

https://wearezeta.atlassian.net/browse/SQSERVICES-1923